syntax = "proto3";

package simulation;

service SimulationService {
  rpc RunSimulation(SimulationRequest) returns (SimulationResponse);
}

message SimulationRequest {
  string scenario_id = 1;
  map<string, string> parameters = 2;
}

message SimulationResponse {
  string status = 1;
  string result = 2;
}
